benchmark/common.js

```diff
@@ -238,20 +238,18 @@ Benchmark.prototype.report = function(value) {
     console.log('%s: %s', heading, value.toFixed(5));
   else if (outputFormat == 'csv')
     console.log('%s,%s', heading, value.toFixed(5));
-
-  process.exit(0);
 };
 
 Benchmark.prototype.getHeading = function() {
   var conf = this.config;
 
   if (outputFormat == 'default') {
     return this._name + ' ' + Object.keys(conf).map(function(key) {
-      return key + '=' + conf[key];
+      return key + '=' + JSON.stringify('' + conf[key]);
     }).join(' ');
   } else if (outputFormat == 'csv') {
     return this._name + ',' + Object.keys(conf).map(function(key) {
-      return conf[key];
+      return JSON.stringify('' + conf[key]);
     }).join(',');
   }
 };
```

Comment:
Either the changes to this file should be a separate commit or the commit log should explain why they are necessary.